docker-compose.yml

```
@@ -24,32 +24,32 @@ services:
-    depends_on:
-      postgres:
-        condition: service_healthy
-      redis:
-        condition: service_healthy
-      kafka:
-        condition: service_healthy
-    deploy:
-      resources:
-        limits:
-          memory: 512m
+#  delivery-service:
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)